src/workerd/api/events.h

```diff
@@ -6,6 +6,18 @@
 
 namespace workerd::api {
 
+class CloseEvent: public Event {
+ public:
+  CloseEvent() = default;
```

Comment:
This can't use the default constructor. You at least need to pass in the `"close"` event name.